ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/200__metrics.robot

```diff
@@ -12,14 +12,12 @@ Test Tags           ExcludeOnODH
 
 
 *** Variables ***
-@{RECORD_GROUPS}    SLOs - MCAD Controller     SLOs - CodeFlare Operator
-...    SLOs - Data Science Pipelines Operator    SLOs - Data Science Pipelines Application
+@{RECORD_GROUPS}    SLOs - Data Science Pipelines Operator    SLOs - Data Science Pipelines Application
 ...    SLOs - Modelmesh Controller
 ...    SLOs - ODH Model Controller
 ...    SLOs - RHODS Operator v2
 
-@{ALERT_GROUPS}     SLOs-probe_success_codeflare    Distributed Workloads CodeFlare
-...    SLOs-haproxy_backend_http_responses_dsp    RHODS Data Science Pipelines    SLOs-probe_success_dsp
+@{ALERT_GROUPS}    SLOs-haproxy_backend_http_responses_dsp    RHODS Data Science Pipelines    SLOs-probe_success_dsp
```

Comment:
Do I understand correctly that we won't `re-add` deleted lines once #1040 is done? If so, I'm okay with this. If not and this is supposed to be temporary, I would prefer commenting it with some explanation instead of deleting.

Reply:
I've modified the description to explain it better. Once 1040, the tests will just verify the rules for components that are enabled